foreman/data_refinery_foreman/surveyor/array_express.py

```diff
@@ -104,19 +111,17 @@ def create_experiment_from_api(self, experiment_accession_code: str) -> Dict:
         try:
             experiment_object = Experiment.objects.get(accession_code=experiment_accession_code)
             logger.debug("Experiment already exists, skipping object creation.",
-                experiment_accession_code=experiment_accession_code,
-                survey_job=self.survey_job.id)
+                         experiment_accession_code=experiment_accession_code,
+                         survey_job=self.survey_job.id)
         except Experiment.DoesNotExist:
 
             experiment_object = Experiment()
             experiment_object.accession_code = experiment_accession_code
             experiment_object.source_url = request_url
             experiment_object.source_database = "ARRAY_EXPRESS"
             experiment_object.name = parsed_json["name"]
-            experiment_object.description = parsed_json["description"][0]["text"]
-            experiment_object.platform_name = experiment["platform_accession_name"]
-            experiment_object.platform_accession_code = experiment["platform_accession_code"]
             experiment_object.technology = "MICROARRAY"
```

Comment:
Can be removed?

Reply:
I was wrong about this. I ended up keeping `technology` on Experiments so they could be filtered on that easily.